nucleus/deploy/client.py

```diff
@@ -109,6 +112,7 @@ def create_model_bundle(
         model: Optional[DeployModel_T] = None,
         load_model_fn: Optional[Callable[[], DeployModel_T]] = None,
         bundle_url: Optional[str] = None,
+        globals_copy: Optional[Dict[str, Any]] = None,
```

Comment:
Is there a reason that the client has to pass in globals().copy(), or can we get the value of that from inside this function? I think if there isn't a reason for the client to pass this in then we shouldn't make them do that

Reply:
to my understanding no... There are some problems with using iterating `globals()` i.e. `for k, v in globals.items()` because size of globals changes while iterating. If you called globals from outside of the function and pass it as `globals_copy` then it's fine.

To answer you question: no it is not needed to `.copy()` it (I think). But I used this name to keep the same naming convention as in https://github.com/scaleapi/nucleus-python-client/blob/hmi-release/nucleus/deploy/find_packages.py#L323